src/main/resources/db/migration/V1_1__init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE users (
+    user_id INTEGER PRIMARY KEY,
+    email TEXT NOT NULL,
+    username TEXT NOT NULL,
+    bio TEXT,
+    image TEXT
+);
+
+CREATE TABLE articles (
+    slug TEXT PRIMARY KEY,
+    title TEXT NOT NULL,
+    description TEXT,
+    body TEXT NOT NULL,
+    created_at INTEGER NOT NULL,
```

Comment:
shouldn't this be `TIMESTAMPTZ`?

Reply:
Looks like sqlite doesn't have separate type for timestamps: https://www.sqlite.org/datatype3.html

Reply:
ah ok, I assume pgsql